plugins/dbgate-plugin-postgres/src/backend/Analyser.js

```diff
@@ -191,6 +206,42 @@ class Analyser extends DatabaseAnalyser {
       columnName: x.column_name,
     }));
 
+    const procedureParameters = routineParametersRows.rows
+      .filter(i => i.routine_type == 'PROCEDURE')
+      .map(i => ({
+        objectId: 'procedures:' + i.specific_schema + '.' + i.routine_name + '@' + i.pure_name,
```

Comment:
i.routine_name will be undefined, this column is not in result of procedureParameters query

ID should be only: 'procedures:' + i.specific_schema + '.' + i.pure_name,

BTW why not to use string interpolation here?